### ISSUE TITLE
Window resize when showing sidebars

### ISSUE THREAD

**Issue by trevormeier** [notaapp/talk#184]
Hi there—trying out Nota and really enjoying it so far!

One annoyance for me is that generally I prefer for the text editing window to stay the same size when I open & close sidebars. As it is now, pressing Ctrl-2 or Ctrl-3 shrinks the text editing window, often to be overly small on my laptop window. I'd prefer it if there was an option in preferences to keep the text editing window the same size when opening and closing sidebars (so the overall window size would grow and shrink as sidebars are opened and closed). This would need some logic built in, for example to not expand beyond the dimensions of the screen.

Thanks for making Nota! I'm going to keep trying it for a few weeks, but I expect it's going to become my new home for markdown.

**Comment by astoilkov** [comment]
Hi. Thanks for the kind words. I hope Nota keeps growing on you.

I think the request is interesting. I should admit I've never heard of such a feature anywhere. Are you used to this from somewhere else?

I have an idea of how I can solve your problem sooner. Have you tried window resizing apps? You can create presets for resizing the Nota window at specific widths and resize it when you show/hide sidebars.

A quick question: You probably prefer to not have Nota maximized in order to see content from other apps in the background when the sidebars are hidden. Am I guessing correctly?

**Comment by trevormeier** [comment]
Yes, I often have Nota open amongst several other creative apps.

Things (from Cultured Code) is an example of sidebars working this way. I believe Nova and IA Writer also work like this.

Sent from ProtonMail for iOS

15, Antonio Stoilkov ***@***.***> escreveu:

**Comment by astoilkov** [comment]
Cool. Thanks for the additional information. We will explore the idea.

**Comment by blumelume** [comment]
I don't think this would be very comfortable for people with window-managers (Amethyst, Rectangle, ...). And I think they are becoming more and more popular.
So personally, I pledge for this to be optional.

Just for your consideration.
